File: 백준/Bronze/17530. Buffoon/Buffoon.java
import java.util.*;
public class Main{
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();
        int arr[] = new int [n];
        int max = 0;
        for(int i = 0; i < n; i++){
            arr[i] = sc.nextInt();
            if(max < arr[i])
                max = arr[i];
        }
        if(arr[0] >= max)
            System.out.println("S");
        else
            System.out.println("N");
    }
}

File: 백준/Bronze/17530. Buffoon/Buffoon.c
#include<stdio.h>
int main(void){
    int n;
    scanf("%d",&n);
    int arr[n];
    int max = 0;
    for(int i = 0; i < n; i++){
        scanf("%d",&arr[i]);
        if(max < arr[i])
            max = arr[i];
    }
    if(arr[0] >= max)
        printf("S\n");
    else
        printf("N\n");
    return 0;
}